homeassistant/components/geofency/device_tracker.py

```diff
@@ -1,5 +1,5 @@
 """
-Support for the Geofency platform.
+This platform provides the device tracker for the Geofency component.
```

Comment:
We should stick to the standard phrasing in the docstring. It starts with "Support...".

Reply:
Got it, changed to:
> Support for the Geofency device tracker platform.